Sources/ComposableArchitecture/Store.swift

```diff
@@ -424,9 +459,16 @@ public final class Store<State, Action> {
   @inline(__always)
   private func threadCheck(status: ThreadCheckStatus) {
```

Comment:
Strictly speaking these should probably be renamed to `queueCheck` and `QueueCheckStatus`.